### DISCUSSION TITLE
Using serenity within global company

### DISCUSSION THREAD

**Discussion by Liqynn** [serenity-is/Serenity#5904] [General]
Hi!
What is the recommended way to use serenity within global company with offices in China, USA, Europe ?
One cloud database with several  application servers ?
Replicated databases and files with corresponded   application servers ?
Anything else?

**Comment by VictorTomaili** [comment]
If you need business support for big projects and/or topics you wrote about, try to contact serenity support via email.

**Reply by VictorTomaili** [reply]
@Liqynn please don't write private email responses here. It's my mistake about the support email address, you need to send an email to sales instead. Support mail is for premium customers. And your questions need these support packages. 
I mean if your company needs an answer to these questions, contact on sales is an option with premium support. Otherwise, community support already gives answers with their customized solutions if they have.

**Reply by Liqynn** [reply]
@VictorTomaili, thanks for your answer.
I have no project for now, I only investigate frameworks. I'll be grateful if community shares with ideas about this.
